sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchClientBuilder.java

```diff
@@ -258,7 +276,7 @@ public SearchClientBuilder httpClient(HttpClient client) {
 
     /**
      * Sets the HTTP pipeline to use for the service client.
-     * <p>
```

Comment:
Don't remove these `<p>` tags, Javadocs doesn't retain text newlines as it is HTML. This without it the Javadocs will go from being:

```
Sets the HTTP pipeline to use for the service client.

If pipeline is set...
```

to

```
Sets the HTTP pipeline to use for the service client. If pipeline is set...
```